Updates/3755_orgrimmar_grunt_path.sql

```diff
@@ -0,0 +1,64 @@
+UPDATE `creature` SET `MovementType`=2 WHERE `guid`=11789;
```

Comment:
If you use linear instead of circular pathing, you can reduce the amount of points by almost 50%.

Reply:
Thanks! I did think about using a linear wp movement, but the path from sniffs has two points that differ between the two "passes". 2/25 and 5/21. I wasn't sure on which one to pick, then settled on the one that used the exact data (although the difference is just a few pixels :P)